sdks/java/testing/load-tests/src/main/java/org/apache/beam/sdk/loadtests/ParDoLoadTest.java

```diff
@@ -62,6 +62,18 @@
     @Description("Options for synthetic step")
     @Validation.Required
     String getStepOptions();
+
+    @Description("Number of reiterations over values to perform.")
+    @Default.Integer(1)
+    Integer getIterations();
```

Comment:
This is not used in the test later, right? 

Actually, according to the proposal, it looks like the whole ParDoLoadTest pipeline needs to be changed first. It doesn't iterate nor it goes over multiple counter (metric counter) operations to test inter-operation / counter operations overhead. 

I'm fine with doing this in a separate PR (leaving only combine tests here) or amend the testing pipeline here (up to you). Let me know if I can help

Reply:
Third option - commit only the configurations (as is now) but make a follow up PR that uses the pipeline options well (it looks like what you wanted to achieve here). In this case please create jira for this

Reply:
I think we should extract this change to another PR and do it later. I'll get right on it.